surrealdb/go.mod

```diff
@@ -0,0 +1,67 @@
+module github.com/gofiber/storage/surrealdb
+
+go 1.23.0
+
+
+require (
+	github.com/stretchr/testify v1.10.0
+	github.com/surrealdb/surrealdb.go v0.3.2
```

Comment:
bug: please run `go mod tidy` in order to sync the file. Also, could you please use testcontainers-go `v0.37.0`? It was released the last week 🙏